lib/puppet/forge.rb

```diff
@@ -90,9 +90,9 @@ def search(term)
   # @see SemanticPuppet::Dependency::Source#fetch
   def fetch(input)
     name = input.tr('/', '-')
-    uri = "/v3/releases?module=#{name}"
+    uri = "/v3/releases?module=#{name}&sort_by=version"
```

Comment:
Out of curiosity, why did you add the sort by query param here?

Reply:
I'll add some info to the commit message.

Reply:
Can you query directly by version number here?

Reply:
Yes but PMT currently collects all the available versions in case it needs them when resolving dependencies.